card-unification.md

```diff
@@ -0,0 +1,100 @@
+# Card Unification
+
+  * [ ] Change card shape to object with `type`, `name`, `render` and optional `edit` properties
+  * [ ] Card's `type` is validated by renderer (dom renderer cannot render 'text', e.g.)
+  * [ ] Change arguments passed by editor to card's `render` (or `edit`) method
+    * [ ] single argument object with `name`, `onTeardown`, `isInEditor`, `options`, `editorEnv` and `payload` properties
+  * [ ] Return value of card's `render` (or `edit`) method is appended/concatenated by renderer
```

Comment:
Looks like the other comments got removed here upon update.  My response:

This doesn't make a huge difference either way, but the idea is to simplify things . Leave the rendering to the actual render, instead of doing it in the card.

 - Saves writing `appendChild`, concatenation, `addSubview` code in every card, and just let's the renderer do the right thing.
 - `appendChild` abstractions can be kept in the renderer instead of re-implementing that in cards
 - The current pattern forces using wrapper dom element for cards.
 - Passing in a buffer element opens easier possibilities of users leaking it

I don't understand the iOS argument.  You return a UIView and the renderer does `addSubview`, instead of writing that line in every card.